### PR TITLE
Sync temperature sensor device type with specs

### PR DESCRIPTION
**Pull request by jadhavrohit924**
#### Problem
What is being fixed?  Examples:
* Temperature sensor device type was not matching specs

#### Change overview
Sync temperature sensor device type with specs

#### Testing
How was this tested? (at least one bullet point required)
* Verified temperature sensor device type syncs with specs

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -1366,22 +1366,6 @@ limitations under the License.
...
-                <requireCommand>ViewGroup</requireCommand>
-                <requireCommand>ViewGroupResponse</requireCommand>
-                <requireCommand>GetGroupMembership</requireCommand>
-                <requireCommand>GetGroupMembershipResponse</requireCommand>
-                <requireCommand>RemoveGroup</requireCommand>
-                <requireCommand>RemoveGroupResponse</requireCommand>
-                <requireCommand>RemoveAllGroups</requireCommand>
-                <requireCommand>AddGroupIfIdentifying</requireCommand>
-            </include>
             <include cluster="Temperature Measurement" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```
Comment (dhrishi) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1369]: Don't you need the required attributes for temperature measurement cluster here?

Comment (jadhavrohit924) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1369]: If attributes mentioned cluster adds all mentioned attributes and if not it adds only mandatory attributs.

**Review by woody-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (dhrishi) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1369]: So you need only the mandatory ones in this device type for this cluster? As per the spec?

Comment (jadhavrohit924) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1369]: Yes

**Review by dhrishi — APPROVED** [review]

**Pull request merged**